<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QMainWindow" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="saveAction"/>
   <addaction name="separator"/>
   <addaction name="rectangleRubberAction"/>
   <addaction name="horizontalRubberAction"/>
   <addaction name="verticalRubberAction"/>
   <addaction name="zoomResetAction"/>
   <addaction name="separator"/>
   <addaction name="configAction"/>
  </widget>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/save.svg</normaloff>:/assets/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="rectangleRubberAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/arrows.svg</normaloff>:/assets/icons/arrows.svg</iconset>
   </property>
   <property name="text">
    <string>rectangleRubber</string>
   </property>
  </action>
  <action name="verticalRubberAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/arrows-v.svg</normaloff>:/assets/icons/arrows-v.svg</iconset>
   </property>
   <property name="text">
    <string>verticalRubber</string>
   </property>
  </action>
  <action name="horizontalRubberAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/arrows-h.svg</normaloff>:/assets/icons/arrows-h.svg</iconset>
   </property>
   <property name="text">
    <string>horizontalRubber</string>
   </property>
  </action>
  <action name="zoomResetAction">
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/undo.svg</normaloff>:/assets/icons/undo.svg</iconset>
   </property>
   <property name="text">
    <string>zoomReset</string>
   </property>
  </action>
  <action name="configAction">
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/settings.svg</normaloff>:/assets/icons/settings.svg</iconset>
   </property>
   <property name="text">
    <string>config</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
